package.json

```diff
@@ -127,6 +127,7 @@
     "react-middle-ellipsis": "^1.1.0",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
+    "react-tilt": "^0.1.4",
```

Comment:
This library is not updated for 3 years and seems not publishing upstream tilt.js change

Reply:
It's stable enough for this little feat.